macros/WebAssemblySidebar.ejs

```diff
@@ -23,30 +23,36 @@ var text = mdn.localStringMap({
 
 <ol>
   <li data-default-state="open"><a href="<%=baseURL%>"><strong><%=text['WebAssembly_home_page']%></strong></a>
-  <li data-default-state="open"><a href="<%=baseURL%>"><%=text['Tutorials']%></a>
-    <ol>
-      <li><a href="<%=baseURL%>/Concepts"><%=text['WebAssembly_concepts']%></a></li>
-      <li><a href="<%=baseURL%>/C_to_wasm"><%=text['Compiling_to_wasm']%></a></li>
-      <li><a href="<%=baseURL%>/Using_the_JavaScript_API"><%=text['JavaScript_API']%></a></li>
-      <li><a href="<%=baseURL%>/Understanding_the_text_format"><%=text['Text_format']%></a></li>
-      <li><a href="<%=baseURL%>/Text_format_to_wasm"><%=text['Text_format_to_wasm']%></a></li>
-      <li><a href="<%=baseURL%>/Loading_and_running"><%=text['Loading_and_running']%></a></li>
-      <li><a href="<%=baseURL%>/Caching_modules"><%=text['Caching_modules']%></a></li>
-      <li><a href="<%=baseURL%>/Exported_functions"><%=text['Exported_functions']%></a></li>
-    </ol>
+  <li class="toggle">
```

Comment:
The ``{{WebAssemblySidebar}}`` macro was correctly updated, as seen on https://developer.mozilla.org/en-US/docs/WebAssembly/C_to_wasm

<img width="464" alt="pr 789 webassemblysidebar" src="https://user-images.githubusercontent.com/286017/45395241-cf597d00-b5fa-11e8-9196-13022268b9b8.png">